.github/actions/pr-title-checker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Copyright 2024 The Kubernetes Authors.
```

Comment:
By last we should either deprecated the Golang code with annotations/comments since we are replacing the old implementation with the shell script.